runbotics/runbotics-scheduler/src/websocket/process-launch/bot-process.service.utils.ts

```diff
@@ -1,6 +1,6 @@
 import { ProcessInstanceStatus } from 'runbotics-common';
 
-const COMMON_PROCESS_INSTANCE_FIELDS = ['input', 'error', 'root_process_instance_id', 'orchestrator_process_instance_id', 'trigger', 'triggered_by', 'step','user_id', 'process_id', 'bot_id'];
+const COMMON_PROCESS_INSTANCE_FIELDS = ['input', 'error', 'root_process_instance_id', 'orchestrator_process_instance_id', 'trigger', 'trigger_data', 'step','user_id', 'process_id', 'bot_id'];
```

Comment:
imo each to new line

Reply:
each one would generate too many lines. I have implemented the compromise